src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -20,6 +22,10 @@
     <ProjectJson>unix/project.json</ProjectJson>
     <ProjectLockJson>unix/project.lock.json</ProjectLockJson>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetGroup)' == 'net46' and '$(ProjectJson)' == '' ">
+    <ProjectJson>net46/project.json</ProjectJson>
+    <ProjectLockJson>net46/project.lock.json</ProjectLockJson>
+  </PropertyGroup>
```

Comment:
The packaging error is caused by the explicit net46 entry below. You should remove both of the net46 PackageDestination items below in this project.